conf/airframes/TUDELFT/tudelft_conf.xml

```diff
@@ -123,6 +123,7 @@
   <aircraft
    name="CX10"
    ac_id="18"
+   ac_id="15"
```

Comment:
this is still not properly rebased on master

Reply:
True, 

does smb know whether it should be 18 or 15?

Reply:
True.

Any idea whether it was 15 or 18?